### ISSUE TITLE
Can't toggle descriptions

### ISSUE THREAD

**Issue by continuitydrift** [continuitydrift/text_adventure#3]
I am trying to get the code to where the first time the player goes into a room, there is a long description, and any time after that there is only a short description of the room. I've tried creating a variable set to false for having visited, but am having trouble getting this to actually change the room's text to the short version. Any ideas?

**Comment by wolfshtein** [comment]
yeah going from scratch again is the best thing you can do, in other words it's verify your plan

:hamburger:
